pkg/config/config.go

```diff
@@ -340,25 +366,51 @@ func (lci *LocalConfigInfo) SetConfiguration(parameter string, value string) (er
 // GetConfiguration uses reflection to get the parameter from the localconfig struct, currently
 // it only searches the componentSettings
 func (lci *LocalConfigInfo) GetConfiguration(parameter string) (interface{}, bool) {
+
+	if strings.ToLower(parameter) == "cpu" {
+		if lci.ComponentSettings.MinCPU == nil {
+			return lci.ComponentSettings.MinCPU, true
+		}
+		return *lci.ComponentSettings.MinCPU, true
+	}
 	return getConfiguration(lci.ComponentSettings, parameter)
 }
 
 // DeleteConfiguration is used to delete config from local odo config
-// TODO: Use reflect to delete parameters
 func (lci *LocalConfigInfo) DeleteConfiguration(parameter string) error {
 	if parameter, ok := asLocallySupportedParameter(parameter); ok {
-
-		switch parameter {
-		case "componenttype":
-			lci.ComponentSettings.ComponentType = nil
+		if parameter == "cpu" {
+			lci.ComponentSettings.MinCPU = nil
+			lci.ComponentSettings.MaxCPU = nil
+		}
+		if err := deleteConfiguration(&lci.ComponentSettings, parameter); err != nil {
+			return err
 		}
-
 		return writeToFile(lci.LocalConfig, lci.Filename)
 	}
 	return errors.Errorf("unknown parameter :'%s' is not a parameter in local odo config", parameter)
 
 }
 
+func deleteConfiguration(info interface{}, parameter string) error {
```

Comment:
Codecov reports this function is missing UTs. Could you please bring some attention here.

How to check missing coverage
1. Go to the coverage report (for example - https://github.com/redhat-developer/odo/pull/1325#issuecomment-463237149)
2. Click on hyperlink ```pkg/config/config.go``` in ```Impacted Files``` table 
3. Then it will be easy to visualize the misses through color coding.

Reply:
added UTs @amitkrout